programFiles/may.cpp

```diff
@@ -0,0 +1,327 @@
+#include<iostream>
+#include<vector>
+#include<fstream>
+#include<string>
+#include<sstream>
+#include<cmath>
+double cRound(double number){
+	if(fmod(number,10)>=5)return (((int)number/1)+1);
+	else return ((int)number/1);
+}
+//N-число строк M-число столбцов
+class matrix{
+		int Nsize;
+		int Msize;
+		std::vector<std::vector<double>> matrixData;
+	public:
+		//Конструкторы. 1) Создание пустой (нулевой) матрицы NxM 2) Считывание матрицы из файла

```

Comment:
In a constructor that reads a matrix from a file, you can use the >> and getline operators to read the coordinates of the matrix from the file. Then the input variable will not be needed.